README.md

```diff
@@ -0,0 +1,60 @@
+# Launch Checklist
+
+## Assignment Requirements
+
+For this "Launch Checklist Form" assignment, you will use a form submission and fetch request to accomplish the following:
+
+1. Validate the user responses with `preventDefault()` to ensure the following: 
+
+   a. The user has entered something for every field.
+   b. The user has entered string values for names and number for fuel and cargo levels.
+
+1. With validation, update a list of what is currently ready or not ready for the shuttle launch.
+1. Use the DOM to update CSS to indicate what is good and bad about the shuttle and whether it is ready for launch.
+1. Fetch some planetary JSON to update the mission destination with vital facts and figures about where the shuttle is headed.
+
+The submitted web page will look similar to the [demo app](http://launch-checklist-launchcodeeducation.s3-website-us-east-1.amazonaws.com/).
+
+## How Your Assignment is Graded
+
+### Grading Tests
+
+You can run the tests locally or check out Github Classroom to see if your work passes the autograding tests. Remember in Github Classroom, if you have passed all the tests, you get a green checkmark and if one or more of the tests fails, then you get a red "x". 
+
+*Note*: Running `npm test` locally will result in one `grading-dom` test failing. This has to do with the asynchronous nature of fetch and should pass in the Github Classroom autograder.
+
+A passing assignment will have an output of: `11 specs, 0 failures`.
+
+If the output matches above, your submission meets the testing requirements! 
+
+### Beyond the Tests
+
+After reviewing the test output, your TA will check your submission for the following:
+
+1. Is the top box populated with the planetary data from the fetch request?
+1. On invalid form submission:
+
+   a. Is the form data persistent? The page should not reload here.
+   b. Does an alert appear if an empty or partially filled form is submitted? Is the alert message relevant?
+   c. Does an alert appear if incorrect data is entered in any field? Names should be strings and fuel and cargo amounts should be numbers.
+
+1. On valid form submission:
+
+   a. Do the pilot and co-pilot values submitted appear as the names in the third box?
+   b. Does the fuel value message reflect if the submitted value is too low (less than 10,000) or in the correct range?
+   c. Does the cargo value message reflect if the submitted value is too high (more than 10,000) or in the correct range?
+   d. Is the header message red and does it read 'Shuttle Not Ready for Launch' when either fuel or cargo are not in range?
+   e. Is the header message green and does it read 'Shuttle is Ready for Launch' when both fuel and cargo are submitted in range?
+
+1. You added JavaScript in ``script.js`` that is enclosed within a load event listener.
+1. A submit event listener handles the form submission.
```

Comment:
Formatting is off when previewing the file.

The second ordered list elements are not on separate lines:
a
b

But all on one line.

ab

Reply:
Done!